easybuild/easyconfigs/a/ABySS/ABySS-1.5.1-ictce-5.4.0-Python-2.7.3.eb

```diff
@@ -0,0 +1,30 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Authors::   Jordi Blasco <jordi.blasco@nesi.org.nz>
+# New Zealand eScience Infrastructure (NeSI)
+
+name = 'ABySS'
+version = '1.5.1'
+versionsuffix = '-Python-2.7.3'
+
+homepage = 'http://www.bcgsc.ca/platform/bioinfo/software/abyss'
+description = """Assembly By Short Sequences - a de novo, parallel, paired-end sequence assembler"""
+
+toolchain = {'name': 'ictce', 'version': '5.4.0'}
+toolchainopts = {'usempi': True}
+
+# eg. http://www.bcgsc.ca/downloads/abyss/abyss-1.3.4.tar.gz
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['https://github.com/bcgsc/abyss/releases/download/%(version)s/']
+buildopts = 'AM_CXXFLAGS=-Wall'
```

Comment:
@jordiblasco: ok, that's just about warnings, ignore. See the other comments above